packages/flutter_tools/templates/plugin/lib/projectName.dart.tmpl

```diff
@@ -6,6 +6,8 @@ class {{pluginDartClass}} {
   static const MethodChannel _channel =
       const MethodChannel('{{projectName}}');
 
-  static Future<String> get platformVersion =>
-      _channel.invokeMethod('getPlatformVersion');
+  static Future<String> get platformVersion async {
+    final String version = await _channel.invokeMethod('getPlatformVersion');
+    return version;
```

Comment:
this looks like the remains of debugging? (as far as i can tell the new code is just a more verbose version of the old code)

Reply:
`invokeMethod` returns `Future`, not `Future<String>` so in Dart 2 the old form does not work.